src/status_im/ui/screens/profile/views.cljs

```diff
@@ -215,8 +215,10 @@
    [react/view styles/settings-item
     [react/text {:style styles/settings-item-text}
      (i18n/label label-kw)]
-    [react/text {:style      styles/settings-item-value
-                 :uppercase? component.styles/uppercase?} value]
+    (when-not (string/blank? value)
+      [react/text {:style           styles/settings-item-value
+                   :number-of-lines 1
+                   :uppercase?      component.styles/uppercase?} value])
```

Comment:
can you put value on the next line please ?

Reply:
it's done. thanks.